nixos/modules/services/desktops/gnome3/gnome-settings-daemon.nix

```diff
@@ -0,0 +1,44 @@
+# GNOME Settings Daemon
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.gnome3.gnome-settings-daemon = {
+
+      enable = mkEnableOption "GNOME Settings Daemon";
+
+    };
+
+  };
+
+
+  ###### implementation
+
+  config = mkIf config.services.gnome3.gnome-settings-daemon.enable {
+
+    environment.systemPackages = [ pkgs.gnome3.gnome-settings-daemon ];
+
+    services.udev.packages = [ pkgs.gnome3.gnome-settings-daemon ];
+
+    security.wrappers.gsd-backlight-helper.source = "${pkgs.gnome3.gnome-settings-daemon}/libexec/gsd-backlight-helper";
+
+    nixpkgs.overlays = [
```

Comment:
I think we probably don't want to be setting overlays in NixOS modules.

Reply:
I was thinking the same thing, but @jtojnar thought it would be an improvement.
Not sure if hydra will cache it and I don't have an details to support against it.